backend/apps/matching/templates/staff/approve_participants.html

```diff
@@ -8,6 +8,7 @@
 
 {% block header %}
 <link rel="stylesheet" href="{% static 'css/form.css' %}">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
@maltezacharias Is this still the way do it?

Reply:
We haven't used CDNs because of the privacy issues throughout the project. As for the CSS, depends. If there is no JS on the page (so far there isn't) yes, that's a possible way. If there is also JS code I tend to load the CSS in there (e.g. `frontend/src/main.js`)